src/scenes/CreateEvent/store.js

```diff
@@ -305,28 +307,40 @@ export default class CreateEventStore {
   }
 
   isValidAddress = async () => {
```

Comment:
needs `@action` decorator since we are calling runInAction?

Reply:
I think the runInAction only means the code it wraps will run as an action,  but the isValidAddress doesn't change any observable, so we can leave it as a normal function.
same as next comments